### PR TITLE
US106147-Update summary-view to use new sdk entity

### PR DESCRIPTION
**Pull request by edwardg8**

### CODE REVIEW

File: components/d2l-enrollment-summary-view/d2l-enrollment-summary-view.js [hunk 1]
```diff
@@ -176,37 +183,11 @@ class D2lEnrollmentSummaryView extends mixinBehaviors([ D2L.PolymerBehaviors.Sir
 		tags.push('About 2 hour 30 minutes');
 		return tags;
 	}
-	_handleEnrollmentResponse(enrollment) {
```
Comment (svanherk) [inline components/d2l-enrollment-summary-view/d2l-enrollment-summary-view.js]: 🎉 Yay code deletion!

Comment (edwardg8) [reply components/d2l-enrollment-summary-view/d2l-enrollment-summary-view.js]: haha yea, looks so clean after having them removed.

Comment (m6jones) [reply components/d2l-enrollment-summary-view/d2l-enrollment-summary-view.js]: I love it!

Comment (edwardg8) [reply components/d2l-enrollment-summary-view/d2l-enrollment-summary-view.js]: I love the SDK entity pattern, making code alot more reusable, and helps prevent duplication and making tests easy too

**Pull request merged**